content/docs/1.2.0/tutorial/get-to-know-iofog.md

```diff
@@ -0,0 +1,117 @@
+
+```console
+$ docker ps --filter "name=iofog"
+
+CONTAINER ID IMAGE                      COMMAND                CREATED            STATUS            PORTS                    NAMES
+baefdede1949 iofog/freeboard:latest     "nginx -g 'daemon of…" 14 seconds ago     Up 13 seconds     0.0.0.0:10102->80/tcp    iofog_BrKHZf9PTcT6yjKcrpnRVBcYPFqxqXxb
+ce6d14551a57 iofog/freeboard-api:latest "node /src/index.js"   17 seconds ago     Up 16 seconds     0.0.0.0:10101->80/tcp    iofog_NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3
+43c00c7c9fc0 iofog/sensors:latest       "node /sensors/index"  20 seconds ago     Up 19 seconds                              iofog_PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb
+5df0afa1018a demo_iofog-agent           "/usr/bin/supervisord" About a minute ago Up About a minute 0.0.0.0:8081->22/tcp     iofog-agent
+7146cd77d400 demo_iofog-controller      "sh -c 'iofog-contro…" About a minute ago Up About a minute 0.0.0.0:51121->51121/tcp iofog-controller
+9e64f5e1afd4 demo_iofog-connector       "sh -c 'service iofo…" About a minute ago Up About a minute 0.0.0.0:53321->8080/tcp  iofog-connector
+```
+
+We can also see that in the PORTS column some of these containers have published [port mappings](https://docs.docker.com/config/containers/container-networking/):
+
+For example, there's a microservice running an instance of [Freeboard](https://github.com/Freeboard/freeboard), an open source visualization dashboard for IoT devices. It's running a web server that serves the dashboard. Go ahead and try it out at http://localhost:10102/?load=dashboard.json
+
+The Freeboard microservice doesn't know it's running locally, so it could just as well be running on real edge node hardware!
+
+## Shell Into Our Containers
+
+Our tutorial environment has three ioFog containers:
+
+- iofog-agent (see [Agents](../agents/overview.html))
+- iofog-controller (see [Controller](../controllers/overview.html))
+- iofog-connector (see [Connector](../connectors/overview.html)
+
+We can think of each of these containers as if they were deployed on separate devices. In production, our Controller is most often running on a cloud server and our Agents are each running on individual edge hardware nodes in the field. The Controller is controlling the Agent the same way it would if the devices were hundreds of miles away, and the Connector can broker communication between any microservices we run.
+
+To interact with our containers we can use the [`docker exec -ti`](https://docs.docker.com/engine/reference/commandline/exec/) command.
+
+#### Agent's Container
+
+Let's start by using the `iofog-agent` CLI to find out its status. Note that the first `iofog-agent` in the following command stands for the container name, and the second `iofog-agent` is the executable wrapped in the container.
+
+```console
+$ docker exec -ti iofog-agent iofog-agent status
```

Comment:
Do we want this to be simply `docker exec -ti iofog-agent sh` and then the CLI commands under it, to give them more experience directly using the CLI on the machine running agent/controller vs execing through contianers always, since in Prod. the Agent won't be in docker container, and same with controller.